Source/Core/Core/HW/WiimoteReal/IONix.cpp

```diff
@@ -15,6 +15,7 @@
 
 namespace WiimoteReal
 {
+extern Wiimote* g_wiimotes[MAX_BBMOTES];
```

Comment:
Please define that in the appropriate .h file and include it here instead. Cross-dependencies across compilation units like this are fragile.